Add favourites feature and library top section config

arielj opened the pull request:
This PR uses a similar pattern used for the feature to hide games but to implement the feature of marking games as Favourites. As discussed in Discord, I also added a configuration in the General Settings to configure what is displayed at the top of the Library (nothing / recently played / favourites).

https://user-images.githubusercontent.com/188464/165879423-c48f2664-5e63-4858-8aab-42150d127bfe.mp4

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

flavioislima reviewed (changes requested):
Tested here and everything works pretty good. ⚔️ 
Two things that I believe could be better:
On the settings, put the new setting above the Max Workers one
Put the Recently played as the default setting since it's the current behavior.

arielj commented:
> Tested here and everything works pretty good. crossed_swords Two things that I believe could be better: On the settings, put the new setting above the Max Workers one Put the Recently played as the default setting since it's the current behavior.

done, I move the option, changed the default and also added translatable strings for the options that I forgot that
